track.py

```diff
@@ -266,6 +293,96 @@ def importSpeedLimitTuples(self, tuples, unit='km/h'):
         checkDataFrame(self.speedLimits, self.length)
 
 
+    def importCurvatureTuples(self, tuples, unitRadiusStart='m', unitRadiusEnd='m', clothoidSamplingInterval=None):
+
+        if not self.lengthOk():
+
+            raise ValueError("Cannot import curvature without a valid track length!")
+        
+        if not unitRadiusStart == unitRadiusEnd:
```

Comment:
I understand that it barely make sense to define different units for radius at start and at end, but since we provide two fields for these two values, it should be supported (i.e., radius at start in m and radius at end in km). 

The convertUnit function will take care of converting both to m. Also the check in line 308 not necessary since it happens already inside the convertUnit function.